src/benchq/resource_estimation/openfermion_re.py

```diff
@@ -174,14 +174,15 @@ def get_double_factorized_qpe_toffoli_and_qubit_cost(
     return total_cost, ancilla_cost
 
 
-def get_double_factorized_block_encoding_toffoli_and_qubit_cost(
+def get_double_factorized_block_encoding_info(
     h1: np.ndarray,
     eri: np.ndarray,
     threshold: float,
     bits_precision_state_prep: int = 10,
     bits_precision_rotation: int = 20,
-) -> Tuple[int, int]:
-    """Get the Toffoli and qubit cost for the double factorized block encoding.
+) -> Tuple[int, int, float]:
+    """Get the Toffoli and qubit cost for the double factorized block encoding as
```

Comment:
Update to:
`"""Get the Toffoli count, qubit cost, and the one-norm for the double factorized block encoding as`